src/target/rp.c

```diff
@@ -155,12 +198,18 @@ static bool rp_flash_write(target_flash_s *f, target_addr_t dest, const void *sr
 
 static bool rp_read_rom_func_table(target *t);
 static bool rp_attach(target *t);
-static void rp_flash_prepare(target *t);
-static void rp_flash_resume(target *t);
+static bool rp_flash_prepare(target *t);
+static bool rp_flash_resume(target *t);
 static void rp_spi_read(target *t, uint16_t command, target_addr_t address, void *buffer, size_t length);
 static uint32_t rp_get_flash_length(target *t);
 static bool rp_mass_erase(target *t);
 
+// Our own implementation of bootloader functions for handling flash chip
+static void __attribute__((unused))rp_flash_connect_internal(target *t);
```

Comment:
There's a missing space after the attribute here, and two lines down. Additionally, as they're both actually unused, it might make sense to either comment them out for now (if you think they have value "just in case"), or remove them entirely. Just seems off to be adding intentionally unused code here.

Reply:
I agree, I left it in the code because I was testing to see how calling this function effected the target, but it seems that it is not needed to be called from our code. The linker will cleanup these unused functions so having them preserved in the code for future reference will not effect the size of the binary, but I can hide them so that the compiler never sees them if you prefer.

Reply:
Fixed!